ansible/roles/logstash/templates/logstash-simple.conf

```diff
@@ -0,0 +1,55 @@
+input {
```

Comment:
@plainolneesh could you please clarify if and how this file is used? I don't seem to see it being referenced from any Ansible tasks, so it would not be getting deployed to the actual nodes. 
- If it is unused, we should delete it from the code base
- If it is used and intended to be deployed to the nodes, you will need a `template` task for deploying these within `ansible/roles/logstash/tasks/main.yml`

Reply:
No, that was there for testing. I've removed it now.